modular_skyrat/modules/singularity_engine/code/particle_accelerator/particle_accelerator.dm

```diff
@@ -1,14 +1,15 @@
-/*Composed of 7 parts :
- 3 Particle Emitters
- 1 Power Box
- 1 Fuel Chamber
- 1 End Cap
- 1 Control computer
- Setup map
-   |EC|
- CC|FC|
-   |PB|
- PE|PE|PE
+/*
+	Composed of 7 parts:
+	3 Particle Emitters
+	1 Power Box
+	1 Fuel Chamber
+	1 End Cap
+	1 Control computer
+	Setup map
+		|EC|
+	CC|FC|
+		|PB|
+	PE|PE|PE
```

Comment:
Somehow something tells me like that's not quite the display we want this to have, unless it displays differently in VSC.